### ISSUE TITLE
upsert example missing

### ISSUE THREAD

**Issue by tony-gutierrez** [Azure/azure-documentdb-node#233]
According to the Toc there is an upsert example here: https://github.com/Azure/azure-documentdb-node/blob/master/samples/DocumentManagement/app.js  but it is skipped in the code.

**Comment by srinathnarayanan** [comment]
Thanks for bringing this to our attention. Would you like to send a PR for this @tony-gutierrez ?

**Comment by tony-gutierrez** [comment]
No, I want microsoft to document their own SDKs so that I don't waste my time trying to imagine how to use the services I pay them for.

**Comment by srinathnarayanan** [comment]
We will update the sample soon. Thanks!
